Task7/Task7/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Task7
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите x:");
+            double x = double.Parse(Console.ReadLine());
+            Console.WriteLine(f(x));
+            Console.ReadKey();
+        }
+        static double f(double x)
```

Comment:
Имя метода принято писать с большой буквы
```suggestion
        static double F(double x)
```